google-cloud-spanner/lib/google/cloud/spanner/convert.rb

```diff
@@ -179,6 +179,8 @@ def grpc_type_for_field field
                 code: :ARRAY,
                 array_element_type: grpc_type_for_field(field.first)
               )
+            elsif :PG_NUMERIC === field
```

Comment:
Don't use `===` to test equality with a symbol. Use `==`.

More idiomatic Ruby would be to replace this entire if-elsif-else clause with a case-when.